pkg/trivyk8s/trivyk8s.go

```diff
@@ -152,8 +154,8 @@ func (c *client) ListArtifacts(ctx context.Context) ([]*artifacts.Artifact, erro
 				}
 			}
 
-			// if excludeOwned is enabled and workload is a built-in workload, we skip it
-			if c.excludeOwned && ownerIsBuiltIn {
+			// if excludeOwned is enabled and workload is a built-in workload and if ownerExists, we skip it
+			if c.excludeOwned && ownerIsBuiltIn && ownerExists {
```

Comment:
do we need this additional variable `ownerExists` ?  this following  should be enough, wdyt ? 
```suggestion
			if c.excludeOwned && ownerIsBuiltIn && len(resource.GetOwnerReferences()) > 0 {
```

Reply:
Done.